examples/webgl_geometry_teapot.html

```diff
@@ -363,6 +368,14 @@
 
 			}
 
+			updateStats();
```

Comment:
What is the point of this? This is a static scene that only re-renders when there is user interaction.

Stats are important if performance is of concern. If you want to add it selectively, that makes sense. I am not sure there is value in including Stats in every example.